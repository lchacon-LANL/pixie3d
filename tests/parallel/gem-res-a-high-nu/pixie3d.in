 &datin
    ilevel   = 0
    test     = t

    nxd      = 64 
    nyd      = 64 
    nzd      = 1

    numerical_grid = f

    xmax     = 12.8
    ymax     = 25.6
    zmax     = 1.

    bcs      = 'def','def','per','per','per','per'

    eta      = 5d-3
    nu       = 5d-2
    dd       = 0d-3
    chi      = .12

    temp_ratio = 5.

    di       = 0d0

    equil    = 'gem'
    coords   = 'car'
    gparams  = 0.5,6.4,0d0,0d0,0d0

    dlambda  = .5
    rshear   = 1.
    vperflow = 0.

    dt       = 1d-2
    tmax     = 1d-1 
    dstep    = 1d-1

    pbz      = 0.1
    prho     = 0d-3
    odd      = f
    nh2      = 1

    cnfactor = 0.5
    sm_flag  = 0

    restart  = f
    timecorr = t
    predictor= f

    source   = f

    precon   = 'id'
    mgtol    = 1d-3
    maxvcyc  = 10
    nsweep   = 4
    precpass = 0

    iguess   = 0

    maxksp   = 20
    maxitgm  = 20
    maxitnwt = 10
    tolgm    = 8d-1
    rtol     = 1d-4
    method   = 1
    mf_eps   = 1d-6

    solenoidal = t
    nc_eom_f   = f
    nc_eom_v   = f

    advect     = 2

    debug    = f
    debug_it = 5

    check_grid = f
 /

 &graphdef
    sel_diag  = 2,10,15,16,17,18,19,20,24,25
    sel_graph = 1,2,3,4,5,6,7,8,9
    dplot = 1d-1
    prof_conf = 1,'x',1,10,1
    cont_conf = 3,'x','y',1,1,1
    iplot = 64
    jplot = 65
    kplot = 1
 /

# B-field code
    pbx      = 0.02454
    pby      = 0.02454

# A code
    pbz      = 0.1

########################
    sel_graph = 1,2,3,4,5,6,7,8,14
    sel_graph = 1,8,14,15,16,17,18,19,20
    sel_graph = 1,8,11,14,-15,17,-18,20,0

    equil    = 'tmcar'
    coords   = 'car'

    maxitnwt = 1
    tolgm    = 1d-4
    rtol     = 1d-3
