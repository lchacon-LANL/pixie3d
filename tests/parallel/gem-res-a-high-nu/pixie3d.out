 
 ***********************************************
 ****** PIXIE3D Extended MHD code, V. 0.9 ******
 ******       Developer: L. Chacon        ******
 ******  Los Alamos National Laboratory   ******
 ******       Los Alamos, NM 87545        ******
 ******           LA-CC 07-005            ******
 ***********************************************
 
 ****************** NOTICES ******************
 Using VECTOR POTENTIAL version
 ****************** NOTICES ******************

  Grid mesh:................   64x   64x    1
  Number of grid levels:....    5,    4,    1
  Preconditioning method:... id        
  Time correction...........   T
  External source...........   F
  Numerical grid............   F
  Advection scheme..........   ZIP    
  TS scheme................. Theta    
 
   itime    time     dt    vx_max vy_max vz_max GMRES  Newton  CN factor
      1    0.010  1.0E-02   0.001  0.000  0.000     3     3     0.500
      2    0.020  1.0E-02   0.001  0.001  0.000     3     3     0.500
      3    0.030  1.0E-02   0.002  0.001  0.000     3     3     0.500
      4    0.040  1.0E-02   0.002  0.002  0.000     3     3     0.500
      5    0.050  1.0E-02   0.003  0.002  0.000     3     3     0.500
      6    0.060  1.0E-02   0.003  0.003  0.000     3     3     0.500
      7    0.070  1.0E-02   0.004  0.003  0.000     3     3     0.500
      8    0.080  1.0E-02   0.005  0.003  0.000     3     3     0.500
      9    0.090  1.0E-02   0.005  0.004  0.000     3     3     0.500
     10    0.100  1.0E-02   0.006  0.004  0.000     3     3     0.500

Final statistics

 itime  Newt/ndt  GMRES/ndt  GMRES/Newt
   10       3.0        3.0        1.0
